nustyle.css

```diff
@@ -68,16 +69,207 @@ article h1 {
   margin: 0;
 }
 
-article h2 {
+.intro h2 {
   padding-top: 0;
   font-family: 'Roboto';
   font-size: 24px;
   font-weight: 700;
 }
 
-article p {
+.intro p {
   padding-right: 2em;
   font-family: 'Poppins';
   font-size: 16px;
   font-weight: 400;
 }
+
+.main-project {
+  color: #1c1a19;
+  padding: 4.5em 1.5em 0 1.5em;
+}
+
+.banner {
+  font-family: "Crete Round";
+  font-size: 40px;
+  font-weight: 400;
+  padding-bottom: 0.7em;
+  margin-bottom: 1.8em;
+  border-bottom: 1px solid #212121;
+}
+
+.main-project h2 {
+  font-family: "Crete Round";
+  font-size: 32px;
+  font-weight: 400;
+}
+
+.main-project p {
+  font-family: "Inter";
+  font-size: 16px;
+  font-weight: 400;
+}
+
+.main-project ul {
+  display: grid;
+  grid-template-columns: repeat(4, 1fr);
+  padding: 0;
+  margin: 0;
+}
+
+.main-project li {
+  padding: 0.7em;
+  margin: 1em 1em 1em 0;
+  justify-content: center;
+  border: 1px solid #8993a4;
+  font-family: "Inter";
+  font-size: 15px;
+  font-weight: 500;
+  list-style: none;
+}
+
+.main-project button {
+  background-color: #ff6b00;
+  width: 124px;
+  height: 48px;
+  border: none;
+  font-family: "Inter";
+  color: #fff;
+  font-size: 17px;
+  font-weight: 700;
+  font-style: normal;
+  text-align: center;
+  letter-spacing: 0.05em;
+}
```

Comment:
- I think it is important for you to add the `hover` or pressed effect on your all your buttons as it is one of the project's requirements.

![buttonsnip](https://user-images.githubusercontent.com/99246359/198648946-0ef47884-bdbc-415b-9a6c-f7187a47cbad.PNG)

You can simply do that by adding the  line of code below 👇🏽  to your CSS file

```
button:hover {
background-color: #FFF5E1;
border: 1px solid #F55800;
color: #F55800;
}

.other-projects button:hover {
   . . .
}
```

What do you think about this?? Besides, it will enhance a pretty user interface 🍒 🍒